appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -405,5 +412,33 @@ public void TextColor(int argb) {
       textColor = argb;
       setAdapterData();
   }
+  
+  /**
+   * Returns the listview's text font Size
+   *
+   * @return text size as an float
+   */
+  @SimpleProperty(
+      description = "The text size of the listview items.",
+      category = PropertyCategory.APPEARANCE)
+  public int TextSize() {
+    return textSize;
+  }
+
+  /**
+   * Specifies the ListView item's text font size
+   *
+   * @param integer value for font size
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_NON_NEGATIVE_FLOAT,
+      defaultValue = Component.FONT_DEFAULT_SIZE + "")
+  @SimpleProperty
+  public void TextSize(float fontSize) {
```

Comment:
You should probably change this argument to int as well (and update the PropertyTypeConstants to integer as well.

-Jeff